app/services/RegistrationService.scala

```diff
@@ -56,19 +56,28 @@ class RegistrationService @Inject()(
     registrationConnector.createRegistration(buildEtmpRegistrationRequest(answers, vrn, commencementDate))
   }
 
+
+  def getRegistration()(implicit hc: HeaderCarrier): Future[DisplayRegistrationResponse] = {
+    registrationConnector.getRegistration()
+  }
```

Comment:
what's the point of this function?

Reply:
Dead code, was using it, but then wasn't. Have deleted it